### DISCUSSION TITLE
/g/home page as default page

### DISCUSSION THREAD

**Discussion by Norsagir** [mealie-recipes/mealie#3108] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I migrated to v1.1.1 and now users who have no account get the login page while entering the simple URL.
In order to get access to public recipes, non-loggued users should access to /g/home URL.

### Please provide a concise description of the feature that would resolve your issue.

Making a setting to have /g/home page directly when entering the base URL, and if you want to log in, click "log in" in the up right corner.

### Please consider and list out some caveats or tradeoffs made in your design decision

I don't see anything to say...

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
there is already an pull request open to implement this: #2772 

We are still in discussion whether we want to go this route or let the user implement this via a reverse proxy.

**Reply by Norsagir** [reply]
Thanks for your message, I read this PR and I thought it was only a redirection when a user is logged in and he is logging off.
